### PR TITLE
SpanKind is missing in HttpServerHandler and HttpClientHandler

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java [hunk 1]
```diff
@@ -112,7 +113,7 @@ public HttpRequestContext handleStart(@Nullable Span parent, C carrier, Q reques
     }
     String spanName = getSpanName(request, extractor);
     SpanBuilder builder = tracer.spanBuilderWithExplicitParent(spanName, parent);
-    Span span = builder.startSpan();
+    Span span = builder.setSpanKind(Kind.CLIENT).startSpan();
```
Comment (bogdandrutu) [inline contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java]: Do we also prefix the SpanName with "Recv." and "Sent." ?

Comment (bogdandrutu) [reply contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java]: Asking because we should not do that.

Comment (rghetia) [reply contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java]: no we don't. I'll add that.

Comment (bogdandrutu) [reply contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java]: Did you see my previous comment. We SHOULD NOT do that.

Comment (rghetia) [reply contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java]: for some reason it was showing only the first comment. Later I saw your comment and I edited my response. Anyway, I am not going to add the prefix.

**Pull request merged**